
const chargerDonnees = async (url) => {
    try {
      const response = await fetch(url);
      if (!response.ok) {
        throw new Error('Erreur lors du chargement des donn√©es');
      }
      const data = await response.json();
      return data;
    } catch (error) {
      throw new Error(error.message);
    }
  };
